README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://github.com/projectriff/charts/workflows/CI/badge.svg)
 
-# projectriff Helm charts (and uncharts)
+# projectriff Installation YAML
 
-Helm charts (and uncharts) to install Istio and riff.
+Installation YAML files for riff (includes Cert-Manager, Istio, Knative, Keda, KPack and Kafka dependencies).
```

Comment:
```suggestion
Release YAML files for riff (includes cert-manager, Istio, Knative, KEDA, kpack and Kafka dependencies).
```

Reply:
I have another issue for removing the riff charts #123